project/docs/credits.md

```diff
@@ -1 +1,25 @@
-{{ credits() }}
+<!-- Template repository: https://github.com/pawamoy/jinja-templates
+     Template path: credits.md
+-->
+
+# Credits
+{% set data = get_credits_data() %}
+These projects were used to build `{{ data['project_name'] }}`. **Thank you!**
```

Comment:
Jinja allows you to access dictionary items as attributes:
```suggestion
These projects were used to build `{{ data.project_name }}`. **Thank you!**
```

Reply:
i will apply that to all data items